src/ethereum/ethPlugin.js

```diff
@@ -74,6 +74,9 @@ export class EthereumPlugin extends CurrencyPlugin {
     const buffer = Buffer.from(strippedPassPhrase, 'hex')
```

Comment:
Is this redundant with the `hextoBuf` call later? Seems like it could be cleaned up.